content/docs/apps/experimental/restricting-users-to-trusted-ip-ranges.md

```diff
@@ -20,5 +20,60 @@ To fully limit access to your apps and services, you must ensure that all users
 
 ## Restricting access to your own applications
 
-To limit requests to your own applications, you can modify your application logic directly. Alternatively, you can [can follow this example](https://github.com/alphagov/re-paas-ip-safelist-service#quick-demo) within cloud.gov to [create a route service](https://docs.cloudfoundry.org/services/route-services.html#introduction) 
-to block all traffic that does not originate from a trusted IP address range.
+Limiting or restricting access for both ingress and egress traffic from your applications is fairly straightforward through a feature called [Application Security Groups](https://docs.cloudfoundry.org/concepts/asg.html), or ASGs. ASGs support white- and black-listing network traffic from given IPv4 ranges.
```

Comment:
I'd rework this paragraph to talk about the two cases... "If you only need to restrict to IPv4 ranges, use ASGs. For IPv6 and other more complex restriction cases (eg limiting particular kinds of requests), you can use a proxy app". Then you can add an anchored section heading at the bottom which gives them a pointer to making a front-end proxy app with the nginx-buildpack.

I believe Peter wrote an example of doing this, btw, so we might even have an something to point them to! See [Slack discussion](https://gsa-tts.slack.com/archives/C0DF6LAMQ/p1553803321013900).